systemd/systemd.go

```diff
@@ -405,16 +406,45 @@ func (l Log) String() string {
 	return fmt.Sprintf("%s %s %s", l.Timestamp(), l.SID(), l.Message())
 }
 
+// UseFuse detects if we should be using squashfuse instead
```

Comment:
Sorry, changed opinion about the proper place. Please see updated comment above.